UCLA Radio/UCLA Radio/NotificationManager.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  NotificationManager.swift
+//  UCLA Radio
+//
+//  Created by Nathan Smith on 12/6/16.
+//  Copyright © 2016 ChrisLaganiere. All rights reserved.
+//
+
+import Foundation
+import UserNotifications
+
+class NotificationManager {
+    
+    static let sharedInstance = NotificationManager()
+    private init() {} // This prevents others from using the default '()' initializer for this class.
+    
+    func requestNotificationPermission(application: UIApplication) {
+        // Configure Notifications
+        // User UNUSerNotificationCenter if >= iOS 10, UIUserNotificationSettings otherwise
+        if #available(iOS 10.0, *) {
+            let center = UNUserNotificationCenter.current()
+            center.requestAuthorization(options: [.alert, .badge, .sound]) { (granted, error) in
+                if granted {
+                    print("Yay!")
+                } else {
+                    print("D'oh")
```

Comment:
Hmm... The line
`print("Error: \(error?.localizedDescription).")`
seems to return nil when the user doesn't allow notifications. Since this isn't what we really want, I've changed it to
`if !granted {
                    print("Error: Notifications not authorized by user.")
}`
I'm not that familiar with error handling in Swift though, so if you know a better way to do this, I'd love to learn!